Sentinel/Classes/EmailSender/MailData.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MailData.swift
+//  Sentinel
+//
+//  Created by Antonijo Bezmalinovic on 22.11.2023..
+//
+
+import Foundation
+
+public struct Attachment {
+    
+    let data: Data?
+    let mimeType: String
+    let fileName: String
+    
+    public init(
+        data: Data?,
+        mimeType: String,
+        fileName: String
+    ) {
+        self.data = data
+        self.mimeType = mimeType
+        self.fileName = fileName
+    }
+}
+
+public struct MailData {
+    
+    let subject: String
+    let message: String
+    let isHTML: Bool
+    let attachments: [Attachment]
+    let toRecipients: [String]
+    let ccRecipients: [String]
+    let bccRecipients: [String]
+    
+    public init(
+        subject: String,
+        message: String,
```

Comment:
If we want to send an empty string, I would go with String? and cast it inside as "" if needed, instead of need to write `""` in the `init`. Just a suggestion (it can be subjective).